easybuild/easyconfigs/g/gtest/gtest-1.7.0-CrayGNU-2015.11.eb

```diff
@@ -0,0 +1,45 @@
+# Built with EasyBuild version 2.5.0 on 2016-01-22_09-08-42
+easyblock = "Tarball"
+
+name = 'gtest'
+version = '1.7.0'
+
+homepage = 'https://code.google.com/p/googletest/'
+description = "Google's framework for writing C++ tests on a variety of platforms"
+
+toolchain = {'name': 'CrayGNU', 'version': '2015.11'}
+
+sources = [SOURCE_ZIP]
+source_urls = ['https://googletest.googlecode.com/files']
+
+sanity_check_paths={
+    'files': ['CMakeLists.txt', ],
+    'dirs': ['include', 'src'],
+}
+
+moduleclass = 'devel'
+
+# Build statistics
```

Comment:
Please also remove these build statistics (should be kept on local builds, it doesn't make sense to upload this to the repo).